extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -407,6 +410,84 @@ angular.module('oppia').component('svgFilenameEditor', {
         };
       };
 
+      var loadGroupedObject = function(objId, obj, groupedObjects) {
+        // Checks if the id starts with 'group' to identify whether the
+        // svg objects are grouped together.
+        if (objId.startsWith('group')) {
+          // The objId is of the form "group" + number.
+          var groupId = parseInt(objId.slice(5));
```

Comment:
How about if we declare a variable for 5 and name it something like `GROUP_ID_PREFIX_LENGTH`.

Reply:
Done.